src/app/pages/signing/signing.component.html

```diff
@@ -176,23 +185,124 @@ <h5>Emojis</h5>
   </div>
 </div>
 
-<!-- Signing Experience  -->
-<div class="modal fade" id="signingExperience" tabindex="-1" role="dialog" aria-labelledby="info-title"
+<!-- Picture not showing up? Modal -->
+<div class="modal fade" id="embed-modal" tabindex="-1" role="dialog" aria-labelledby="embed-modal-title"
      aria-hidden="true">
   <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content shadow-lg">
+      <div class="modal-header">
+        <h5 class="modal-title" id="embed-modal-title">Can't picture your perfect message?</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <p>In order for us to display your image, we need something called an <b>embed link</b>. This embed link isn't
+          always easy to come by, depending on where you're trying to get the image from.</p>
+        <p>Not to worry, though! In a couple minutes you can follow the steps below to get the 'embed link' for most
+          photos that you see on the internet!</p>
+        <ol>
+          <li>Upload your photo to some service like Google Drive or Imgur. Or just find a photo that's already
+            online.
+          </li>
+          <li>Open the photo in your browser and see if you can right click the photo and click 'Open Image in New
+            Tab'
+          </li>
+          <li>Keep doing this until the website that you are on contains ONLY the image you want to embed, with no other
+            buttons or logos
+            or anything around it. Once you see this sort of page, then copy the URL as that URL will now work as an
+            'embed link.'
+            If you don't
+            do this step, your link may not display your photo in your message properly. For example,
+            <a target="_blank" href="https://bit.ly/2NRCMvu">this link</a> will not
+            work but <a target="_blank" href="https://bit.ly/2BuNxkA">this one</a> will (and you can get from one to the
+            other by using the 'Open Image in New Tab' technique just mentioned!).
+          </li>
+          <li>If you are not able to do the 'Open Image in New Tab' method for some reason (e.g. notably Google Drive
+            doesn't let you do this), then you will have to research 'how to get the embed link' for whatever service
+            (e.g. Google Drive) that you are trying to get the embed link from.
+          </li>
+        </ol>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Embed Photo Modal -->
```

Comment:
Try to make the vertical spacing even for mobile
![image](https://user-images.githubusercontent.com/39015447/87462049-530c9800-c5dd-11ea-9415-998060973a95.png)